packages/falcon-server/src/schemaDirectives/GraphQLCacheDirective.test.ts

```diff
@@ -18,8 +18,8 @@ const schemaDirectives = {
 };
 
 describe('@cache directive', () => {
-  let cacheProvider;
-  let cache;
+  let cacheProvider: InMemoryLRUCache;
```

Comment:
correct me if I wrong but `InMemoryLRUCache` is a specific implementation of cache provider, am I right?
So here it should be :
```suggestion
  let cacheProvider: KeyValueCache<string>;
```